### PR TITLE
Add fastboot-location-test back

### PR DESCRIPTION
**Pull request by josemarluedke**
This test was removed in #398, but we need to have tests for these use cases.

It seems all tests fails because we are passing a rootURL and that doesn't seem to work at the moment.

I would love someone to pick this up and fix these tests.

cc/ @kratiahuja

### CODE REVIEW

**Comment by josemarluedke** [comment]
I think these tests catch the issue of #441.

**Comment by kratiahuja** [comment]
@josemarluedke I'll work on #441 tonight. Would appreciate if you can pick this up and fix anything broken on top of #441.

**Comment by kratiahuja** [comment]
PR to fix `rootUrl` with `ember s`: https://github.com/ember-fastboot/ember-cli-fastboot/pull/443

File: test/fastboot-location-test.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    });
+  });
+
+  after(function() {
+    return app.stopServer();
+  });
+
+  it('should NOT redirect when no transition is called', function() {
+    return get({
+      url: 'http://localhost:49741/my-roo/test-passed',
```
Comment (kratiahuja) [inline test/fastboot-location-test.js]: There is a typo here.

`my-roo` -> `my-root`

**Comment by josemarluedke** [comment]
Rebased over #443 and tests are now passing.

**Comment by kratiahuja** [comment]
@josemarluedke I'll double confirm later in the evening if there are any missing tests. If not, I'll merge this. Does that sound good?

**Comment by kratiahuja** [comment]
@josemarluedke I think we also need to port the `fastboot-location-config-test` as well. Can you please port them as well: https://github.com/ember-fastboot/ember-cli-fastboot/pull/398/files#diff-259b70bc79b725f43463bf8d2136830d ?

**Comment by kratiahuja** [comment]
Nvm, I see the other PR brings back `fastboot-location-configuration-test` there. I'll merge this. Thanks @josemarluedke

**Pull request merged**